src/framework/preferences/html/PrefsEditorTemplate-lineSpace.html

```diff
@@ -1,5 +1,6 @@
 <div class="flc-prefsEditor-line-space">
     <h2><span class="fl-icon-line-space"></span><label class="flc-prefsEditor-line-space-label" for="line-space"></label></h2>
```

Comment:
This greatly messes the styling. Wondering if it would be okay to leave them in for now as they will probably be used when more semantic descriptions are added for this jira https://issues.fluidproject.org/browse/FLUID-5417